Sources/XcodeGenCLI/GenerateCommand.swift

```diff
@@ -17,6 +17,13 @@ class GenerateCommand: Command {
         defaultValue: false
     )
 
+    let disableVariableExpansion = Flag(
+        "-n",
+        "--no-expansion",
+        description: "Disable variables expansions",
```

Comment:
This only affects `ENV` expansion not all variables, so should probably say `Disable environment variable expansions` as `--no-env`

Reply:
This could also be added under `Added` in the changelog